### PR TITLE
feat: wallet UI claim rewards

### PR DESCRIPTION
**Pull request by namgold**
# New Wallet UI
- new Wallet UI for claiming reward: voting reward and gas refund reward
  + refactor claiming reward hooks to be reuseable
- integrate gas refund cycle API
- update texts & images change
- refactor Icon sprite: add type
- fix Solana loop call on load page init
- add Mixpanel event for gas refund
- fix BE's `gasRefundPerCentage` -> `gasRefundPercentage`
- fix gas refund claim transaction alert & transaction informations
- fix wallet popup restrict to only able to drag inside of screen
- reload claim reward info after claimed
- refactor gas refund use swr => rtk
- fix remove some duplicated rtk middleware

### CODE REVIEW

File: src/components/Announcement/Popups/TransactionPopup.tsx [hunk 1]
```diff
@@ -166,6 +166,7 @@ const SUMMARY: { [type in TRANSACTION_TYPE]: SummaryFunction } = {
   [TRANSACTION_TYPE.CANCEL_LIMIT_ORDER]: summaryCancelLimitOrder,
   [TRANSACTION_TYPE.TRANSFER_TOKEN]: summaryTransferToken,
 
+  [TRANSACTION_TYPE.KYBERDAO_CLAIM_GAS_REFUND]: summaryClaim, //todo namgold
```
Comment (nguyenhoaidanh) [inline src/components/Announcement/Popups/TransactionPopup.tsx:169]: todo

File: src/components/WalletPopup/RewardCenter.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                >
+                  <BalanceTitle>
+                    <Trans>Total Available Rewards</Trans>
+                  </BalanceTitle>
+                </MouseoverTooltip>
+              </Flex>
+
+              <BalanceValue>{formatNumberWithPrecisionRange(totalReward.knc, 0, 8)} KNC</BalanceValue>
+              <Text fontSize={12} fontWeight={500} lineHeight="16px" color={theme.subText}>
+                {totalReward.usd ? `~ $${formatNumberWithPrecisionRange(totalReward.usd, 0, 8)}` : '$ --'}
```
Comment (nguyenhoaidanh) [inline src/components/WalletPopup/RewardCenter.tsx:126]: how about 0

File: src/components/WalletPopup/Transactions/TransactionItem.tsx [hunk 3]
```diff
@@ -342,13 +342,14 @@ const DESCRIPTION_MAP: {
...
 
-  [TRANSACTION_TYPE.KYBERDAO_CLAIM]: Description1Token,
-
   [TRANSACTION_TYPE.APPROVE]: DescriptionApproveClaim,
   [TRANSACTION_TYPE.CLAIM_REWARD]: DescriptionApproveClaim,
 
   [TRANSACTION_TYPE.KYBERDAO_STAKE]: DescriptionKyberDaoStake,
   [TRANSACTION_TYPE.KYBERDAO_UNSTAKE]: DescriptionKyberDaoStake,
+  [TRANSACTION_TYPE.KYBERDAO_CLAIM]: Description1Token,
+  [TRANSACTION_TYPE.KYBERDAO_CLAIM_GAS_REFUND]: Description1Token, //todo namgold: test this
```
Comment (nguyenhoaidanh) [inline src/components/WalletPopup/Transactions/TransactionItem.tsx:351]: test

File: src/pages/KyberDAO/StakeKNC/StakeKNCComponent.tsx [hunk 4]
```diff
@@ -374,14 +383,24 @@ export default function StakeKNCComponent() {
           setAttemptingTxn(false)
           setTxHash(tx)
           setDelegateAddress('')
+          refetchGasRefundInfo()
```
Comment (nguyenhoaidanh) [inline src/pages/KyberDAO/StakeKNC/StakeKNCComponent.tsx:386]: seem duplicate
make a function
```
refresh = ()=>{
setAttemptingTxn(false)
           setAttemptingTxn(false)
          setDelegateAddress('')
          refetchGasRefundInfo()
}
```

File: .env.adpr [hunk 5]
```diff
@@ -1,4 +1,4 @@
-VITE_AGGREGATOR_API=https://aggregator-api.dev.kyberengineering.io
```
Comment (nguyenhoaidanh) [inline .env.adpr:1]: todo

File: src/pages/KyberDAO/StakeKNC/StakeKNCComponent.tsx (hunk 4 above)
Comment (namgold) [reply src/pages/KyberDAO/StakeKNC/StakeKNCComponent.tsx:386]: It's existing code. We should not touch what is working well =))

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**